src/controllers/features/update-acsp/whereDoYouLiveController.ts

```diff
@@ -0,0 +1,74 @@
+import { NextFunction, Request, Response } from "express";
+import { validationResult } from "express-validator";
+import countryList from "../../../../lib/countryList";
+import { formatValidationError, getPageProperties } from "../../../validation/validation";
+import * as config from "../../../config";
+import { Session } from "@companieshouse/node-session-handler";
+import { UPDATE_WHERE_DO_YOU_LIVE, UPDATE_ACSP_DETAILS_BASE_URL, UPDATE_ACSP_CHANGE_DETAILS } from "../../../types/pageURL";
+import { selectLang, addLangToUrl, getLocalesService, getLocaleInfo } from "../../../utils/localise";
+import { saveDataInSession } from "../../../common/__utils/sessionHelper";
+import { AcspData } from "@companieshouse/api-sdk-node/dist/services/acsp";
+import { WhereDoYouLivBodyService } from "../../../services/where-do-you-live/whereDoYouLive";
+import { USER_DATA } from "../../../common/__utils/constants";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+    const lang = selectLang(req.query.lang);
+    const locales = getLocalesService();
+    const session: Session = req.session as any as Session;
+    const acspData: AcspData = session.getExtraData(USER_DATA) ? session.getExtraData(USER_DATA)! : {};
```

Comment:
Why do we need acspData to have some value?  I believe we can just do this.
const acspData: AcspData = session?.getExtraData(USER_DATA)!;

And handle the null of acspData separately

Reply:
Can we please add a comment in the code and also raise a JIRA ticket as as tech debt to do this later.